stix2elevator/convert_stix.py

```diff
@@ -1618,6 +1667,74 @@ def negate_indicator(indicator):
     return hasattr(indicator, "negate") and indicator.negate
 
 
+def find_first_seen_sighting(sightings):
+    earlist_date = sightings[0].timestamp
+    for s in sightings:
+        if earlist_date:
+            if s.timestamp:
+                if s.timestamp <= earlist_date:
+                    earlist_date = s.timestamp
+        else:
+            earlist_date = s.timestamp
+    return earlist_date
```

Comment:
I think this function's contents could be replaced with 
```suggestion
    return min(sightings, key=attrgetter("timestamp")).timestamp
```

as long as this import is added:
```
from operator import attrgetter
```